extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -1063,3 +1063,138 @@ async def get(self, objectId: str = None):
+                application/json:
+                  schema:
+                    allOf:
+                      - $ref: '#/components/schemas/Success'
+                      - type: object
+                        properties:
+                          data:
+                            type: array
+                            items:
+                              type: object
+            400:
+              content:
+                application/json:
+                  schema: Error
+        """
+        ra = self.get_query_argument("ra", None)
+        dec = self.get_query_argument("dec", None)
+        radius = self.get_query_argument("radius", None)
+        radius_units = self.get_query_argument("radius_units", None)
+        if ra is None:
+            return self.error("Missing required parameter: ra")
+        if dec is None:
+            return self.error("Missing required parameter: dec")
+        if radius is None:
+            return self.error("Missing required parameter: radius")
+        if radius_units is None:
+            return self.error("Missing required parameter: radius_units")
+        if radius_units not in ["deg", "arcmin", "arcsec"]:
+            return self.error(
+                "Invalid radius_units value. Must be one of either "
+                "'deg', 'arcmin' or 'arcsec'."
+            )
+        try:
+            ra = float(ra)
+            dec = float(dec)
+            radius = float(radius)
+        except ValueError:
+            return self.error("Invalid (non-float) value provided.")
+        streams = self.get_user_streams()
+
+        # allow access to public data only by default
+        selector = {1}
+
+        for stream in streams:
+            if "ztf" in stream.name.lower():
+                selector.update(set(stream.altdata.get("selector", [])))
+
+        selector = list(selector)
+
+        query = {
+            "query_type": "near",
+            "query": {
+                "max_distance": radius,
+                "distance_units": radius_units,
+                "radec": {"query_coords": [ra, dec]},
+                "catalogs": {
+                    "ZTF_alerts": {
+                        "filter": {},
+                        "projection": {
+                            "objectId": 1,
+                            "candid": {"$toString": "$candid"},
+                            "candidate.ra": 1,
+                            "candidate.dec": 1,
+                            "candidate.jd": 1,
+                            "candidate.fid": 1,
+                            "candidate.magpsf": 1,
+                            "candidate.sigmapsf": 1,
+                            "candidate.drb": 1,
+                            "candidate.programid": 1,
+                            "classifications": 1,
+                            "_id": 0,
+                        },
+                    }
+                },
+            },
+            "kwargs": {
+                "limit": 1000,
```

Comment:
Thinking about this, since we're essentially limiting the alert contents to the size of ~20 floats, I think we can safely bump the limit up to 10000.
```suggestion
                "limit": 10000,
```

Reply:
@acrellin Thinking about this a bit more, could you please add a tooltip to the search box saying that the max radius is 1 deg and the results will be limited to 10,000 nearest matches?